fds/services/fds_service.py

```diff
@@ -97,3 +97,37 @@ def commit(self, message: str, yes: bool):
             self.printer.error(str(e))
             self.printer.error("Git commit failed to execute")
             sys.exit(1)
+
+    def push(self, git_remote: str, dvc_remote: str, ref: str = None):
+        """
+        fds push
+        """
+        try:
+            self.dvc_service.push(remote=dvc_remote)
+            self.printer.warn("Successfully pushed to DVC remote")
+        except Exception as e:
+            self.printer.error(str(e))
+            self.printer.error("DVC push failed to execute")
+            sys.exit(1)
```

Comment:
This and all other `sys.exit` calls should be removed - e.g. if you now run `fds save` and an any stage fails, then (I think) the "fds save failed" print will not appear.
I guess we can have this as a separate issue since it's leftover from earlier.

Reply:
#38